features/src/autogluon/features/generators/drop_duplicates.py

```diff
@@ -34,7 +34,7 @@ class DropDuplicatesFeatureGenerator(AbstractFeatureGenerator):
         Refer to :class:`AbstractFeatureGenerator` documentation for details on valid key word arguments.
     """
 
-    def __init__(self, sample_size_init=500, sample_size_final=2000, **kwargs):
+    def __init__(self, sample_size_init=500, sample_size_final=3000, **kwargs):
```

Comment:
Sampling with replacement may lead to duplicates, so we increase the sample size to compensate for that.